azure-pipelines.yml

```diff
@@ -183,23 +188,23 @@ jobs:
 # registry. Its dependencies should be updated to include all of the
 # official builds if we add more platform/arch combinations.
 
-# TODO: Enable publish to BAR
-#- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-#  - template: /eng/common/templates/phases/publish-build-assets.yml@arcade
-#- phase: publish_bar
-#  displayName: publish to BAR (empty currently)
-#  queue:
-#    name: Hosted VS2017
-#  dependsOn:
-#  - build_Linux_x64_release
-#  - build_OSX_x64_release
-#  - build_Windows_NT_x64_release
-#  - build_Windows_NT_x86_release
-#  - build_Windows_NT_arm_release
-#  - build_Windows_NT_arm64_release
-  # TODO: enable these builds
-  #  - build_rhel_x64_release
-  #  - build_alpine_x64_release
-  #  - build_crossbuild_arm_release
-  #  - build_crossbuild_arm64_release
-
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}:
+  - template: /eng/common/templates/job/publish-build-assets.yml
+    parameters:
+      configuration: Release
+      pool:
+        name: dotnet-internal-temp
+      dependsOn:
+      - build_Linux_arm_release
+      - build_Linux_arm64_release
+      # TODO: depend on musl job once the glibc container issue is fixed
```

Comment:
This issue is fixed. We're already using this in corefx, I added an alpine container with node built from source that works with Azure Dev Ops. 

microsoft/dotnet-buildtools-prereqs:alpine-3.6-WithNode-f4d3fe3-20181213005010

cc: @RussKeldorph